packs/slack/sensors/slack_sensor.py

```diff
@@ -35,7 +36,8 @@ def setup(self):
             msg = 'Failed to connect to the Slack API. Invalid token?'
             raise Exception(msg)
 
-        self._populate_cache(user_data=data['users'], channel_data=data['channels'])
+        self._populate_cache(user_data=self._api_call('users.list'),
```

Comment:
The API must have changed. These attributes are no longer available. But, this seeds the cache and they are resolved with https://github.com/StackStorm/st2contrib/blob/slack-updates/packs/slack/sensors/slack_sensor.py#L111-L123 and put into cache.